services/openuserjs/openuserjs-license.service.js

```diff
@@ -0,0 +1,27 @@
+import { renderLicenseBadge } from '../licenses.js'
+import BaseOpenUserJSService from './openuserjs-base.js'
+
+export default class OpenUserJSLicense extends BaseOpenUserJSService {
+  static category = 'license'
+  static route = { base: 'openuserjs/l', pattern: ':author/:scriptName' }
+
+  static examples = [
+    {
+      title: 'OpenUserJS',
+      namedParams: {
+        author: 'NatoBoram',
+        scriptName: 'YouTube_Comment_Blacklist',
+      },
+      staticPreview: renderLicenseBadge({ licenses: ['GPL-3.0-or-later'] }),
+    },
+  ]
+
+  static defaultBadgeData = { label: 'license' }
+
+  async handle({ author, scriptName }) {
+    const data = await this.fetch({ author, scriptName })
+    return renderLicenseBadge({
+      licenses: [data.UserScript.license[0].value],
```

Comment:
You do realize this is probably incorrect. All Userscript key values are always LIFO. If you are on the site you may read about it specifically when adding a script with a primary license under dual or more licensing.

OUJS Admin

Reply:
Thanks, do you have an example of what that might look like? I haven't seen any with multiple values.

I'm assuming it would be an array with each entry having a `value` key?

Reply:
> do you have an example of what that might look like?

See https://openuserjs.org/about/Frequently-Asked-Questions#q-does-openuserjs-org-have-meta- and the Unit Test script mentioned and also the human readable *(text stream)* at https://openuserjs.org/user/add/scripts *(you need to be logged in to see it)*.